gensim/models/wrappers/ldamallet.py

```diff
@@ -346,3 +347,23 @@ def read_doctopics(self, fname, eps=1e-6, renorm=True):
                     if total_weight:
                         doc = [(id_, float(weight) / total_weight) for id_, weight in doc]
                 yield doc
+
+
+def malletmodel2ldamodel(mallet_model):
+    """
+    Function to convert mallet model to gensim LdaModel. This works by copying the
+    training model weights (alpha, beta...) from a trained mallet model into the
+    gensim model.
+
+    Args:
+    ----
+    mallet_model : Trained mallet model
+
+    Returns:
+    -------
+    model_gensim : LdaModel instance; copied gensim LdaModel
+    """
+    model_gensim = LdaModel(id2word=mallet_model.id2word, num_topics=mallet_model.num_topics,
+                            alpha=mallet_model.alpha, iterations=mallet_model.iterations)
```

Comment:
Yes, probably leave the default. Or give an option (fnc args) to the user to set `gamma_threshold`/`iterations` explicitly.